src/Components/CardboardModal/CardboardModal.types.ts

```diff
@@ -0,0 +1,65 @@
+import {
+    IButtonProps,
+    IButtonStyles,
+    IIconStyles,
+    IModalStyles,
+    IStackStyles,
+    IStyle,
+    IStyleFunctionOrObject,
+    ITheme
+} from '@fluentui/react';
+import { CardboardIconNames } from '../../Models/Constants';
+
+export interface ICardboardModalProps {
```

Comment:
do you think we should add targetRef to keep track of the element which opened that dialog? I kind of remember cases that we needed to trace back the element to focus back or something on dismiss the dialog (probably for accessibility)?

Reply:
hmmm. I could add that if you can point me to where we do it today. I haven't noticed that before.

Reply:
hmm nvm, looks like fluent does that by itself

Reply:
not sure why we needed it but worth to check: 
![image](https://user-images.githubusercontent.com/45217314/193330111-59aa9f5a-5a87-4ac4-99b5-8d4d26924a2e.png)

Reply:
oh gotcha. You can still do that though right? Cause this is handled in the onDismiss function in the caller? It's a ref to something outside the modal.

Reply:
yeah, that is true! :)